### ISSUE TITLE
Edit content field and remove type field from commofpract.yml credits file

### ISSUE THREAD

**Issue by tamara-snyder** [hackforla/website#2816]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, I want to edit the content field and remove the type field from a credit's yml file so that redundant code is removed and the code is easier to understand.

### Details
Currently, in each credit's yml file, there is a `content` field and a `type` field, which contains similar information. Since the fields are redundant, we decided to remove the `type` field.  

Also, we changed the `content` field to the `content-type` field to make it clearer what its purpose is. The possible values for `content-type` field are image, video, or audio; other types of content can be added if necessary. In the future, this will allow developers a way to differentiate easily between different types of content in order to show each credit's media file correctly on the website.

### Action Items
For the file `_data/internal/credits/commofpract.yml`, do the following:
- [x] Change line 4 for the content field from
  ```
  content: icon
  ```
  to
  ```
  content-type: image
  ```
- [x] Remove line 11 for the type field
  ```
  type: icon
  ```
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #2775 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #2775 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is tracked in the epic #2775.

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_data/internal/credits/commofpract.yml`

- Click on the links below to see the source file code:
https://github.com/hackforla/website/blob/e980b70efb40f63c7c1d54595c0bc3b1ab0d3569/_data/internal/credits/commofpract.yml#L4
https://github.com/hackforla/website/blob/e980b70efb40f63c7c1d54595c0bc3b1ab0d3569/_data/internal/credits/commofpract.yml#L11

**Comment by Mattre7** [comment]
i. Available most of the time except for friday-sunday nights, as well as tuesday from 11-5

ii. Finished the issue fairly quickly, just added, committed, and pushed my changes. In the future I will be creating a new branch for the issue I am working on instead of just making the changes on my local master branch. I am hoping this can be reviewed so I can see if I pushed the changes correctly, and if not, what I did wrong and what I can do in the future to do it correctly.

**Comment by ExperimentsInHonesty** [comment]
@Mattre7 the last update we had from you, you said

> i. Available most of the time except for friday-sunday nights, as well as tuesday from 11-5

>ii. Finished the issue fairly quickly, just added, committed, and pushed my changes. In the future I will be creating a new branch for the issue I am working on instead of just making the changes on my local master branch. I am hoping this can be reviewed so I can see if I pushed the changes correctly, and if not, what I did wrong and what I can do in the future to do it correctly.

I am checking to see what happened.  We don't see a PR

**Comment by Mattre7** [comment]
Hi Bonnie! I am so sorry it has been a while since I tended to this issue. A little while ago I got some help from some leads in a meeting and they helped me create a branch specifically for the issue I was working on (I had just been working on my main branch from my fork). Originally I thought I had created a pull request but then realized I hadn't. Still refamiliarizing myself with this process, but I just created the pull request, so I am hoping you are able to see it now.

**Comment by Mattre7** [comment]
Leaving an update on this page. Ended up closing my first pull request and starting from scratch due to quite a few merge conflicts due to an outdated fork/branch of the website. Recently opened a pull request where the only conflicts are the changes I was tasked with on the issue, has already been approved by one, waiting for further approval.
